.github/workflows/builds.yml

```diff
@@ -236,7 +236,8 @@ jobs:
       - name: Python level tests
         shell: bash
         run: >
-          /usr/local/bin/geant4-config --install-datasets
+          patch /usr/local/bin/geant4-config CI/geant4-config.patch
```

Comment:
`CI/geant4-config.patch` is missing not pushed yet right?

Reply:
dang :D

Reply:
It's there now.